spyder/api/panel.py

```diff
@@ -132,9 +134,23 @@ def paintEvent(self, event):
                 self.palette().windowText().color()))
             painter = QPainter(self)
             painter.fillRect(event.rect(), self._background_brush)
+            # Paint cell dividers in the visible region
         else:
             logger.debug(f'paintEvent method must be defined in {self}')
 
+    def paint_cell(self, painter):
```

Comment:
Lets add here a docstring for this method, maybe something like:
```suggestion
    def paint_cell(self, painter):
        """Paint cell dividers in the visible region if needed."""
```